src/test/java/Homework24.java

```diff
@@ -0,0 +1,62 @@
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+import pages.PlaylistPage;
+
+public class Homework24 extends BaseTest{
+    HomePage homePage;
+    PlaylistPage playlistPage;
+    LoginPage loginPage;
+    String newName = "Updated playlist \"Rename.\"";
+    String namedPlaylist = "Created playlist \"Test Pro.\"";
+    String notification = "Deleted playlist \"Test Pro.\"";
```

Comment:
But I have some suggestions to make the script run more stable,
At test `deletePlaylist`, you should not hardcode the name of the playlist to delete, because when running, the playname is not always `Test Pro`. 
nstead, after clickingPlaylist, get that playlist name and verify notification with the playlist name of that clicked delete playlist
// Example:
![image](https://github.com/acl26/A49-QA-Automation-Framework/assets/144403954/2d5e7b3b-b4e1-4fa4-94ed-9a8d7f00e68b)